src/frontend/src/components/Instruction.jsx

```diff
@@ -117,6 +117,17 @@ function Instruction() {
                   using the <b>Save Annotation</b> button.
                 </p>
               </li>
+              <li>
+                <strong className="download-annotation">
+                  Download Annotation
+                </strong>
```

Comment:
I'd like the UI buttons to reflect a little more closely what's actually going on here. You're not downloading the annotations themselves (h3 polygons), but rather samples from them. I think this is a slightly better name for this button, but am open to other ideas!
```suggestion
                <strong className="sample-annotation">
                  Sample Annotation
                </strong>
```